x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/job_creator.ts

```diff
@@ -243,6 +243,25 @@ export class JobCreator {
     );
   }
 
+  public set annotations(enable: boolean) {
```

Comment:
this would be better named `modelPlotAnnotations`

Reply:
Changed it to `modelChangeAnnotations` in https://github.com/elastic/kibana/pull/70678/commits/1b54be9b39e990830c85f2506d040e803725dad5